devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -884,6 +886,21 @@ bool WholeBodyDynamicsDevice::loadSettingsFromConfig(os::Searchable& config)
     yarp::os::Property prop;
     prop.fromString(config.toString().c_str());
 
+    if (prop.check("devicePeriodInSeconds"))
+    {
+	if(!prop.find("devicePeriodInSeconds").isDouble())
+	{
+	    yError() << "wholeBodyDynamics : The devicePeriodInSeconds must be a double";
+	    return false;
+	}
+        this->setPeriod(prop.find("devicePeriodInSeconds").asDouble());
+    }
+    else
+    {
+	yError() << "wholeBodyDynamics : The devicePeriodInSeconds parameter is not found the default one is used. Period: " << this->getPeriod() << " seconds.";
```

Comment:
If it does not prevent the module to start, I would demote this message to ``yWarning``

Reply:
https://github.com/robotology/whole-body-estimators/pull/130/commits/01dd7fdebc6c664b114cfcd299016c1dad281501